lib/index.ts

```diff
@@ -67,7 +67,7 @@ function tryExtractDigestHash(evt: {
 		return evt.aux.Digest;
 	}
 	if (typeof evt.status === 'string') {
-		const matchPull = evt.status.match(/^Digest:\s([a-zA-Z0-9]+:[a-f0-9]+)$/);
+		const matchPull = evt.status.match(/[Dd]igest:\s([a-zA-Z0-9]+:[a-f0-9]+)(\s.+)?$/);
```

Comment:
How about updating it to something like the following, so that the regexp doesn't have to match the whole line?
```suggestion
		const matchPull = evt.status.match(/[Dd]igest:\s([a-zA-Z0-9]+:[a-f0-9]+)[\s$]/);
```

Reply:
Yup, too used to matching as much as possible. Changed!